src/plugins/engagement/common/index.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const PLUGIN_ID = 'engagement';
+export const PLUGIN_NAME = 'engagement';
```

Comment:
NIT: redundant? A plugin has no `name` concept, why two constants?

Reply:
This was boilerplate, produced by the plugin generation script.